browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -5046,6 +5111,26 @@ index 6ffc83c012ffe095046414051161d7506af192a1..8b5095998153f0c3a5762022459dbd09
      Timer m_hoverTimer;
      bool m_hasScheduledCursorUpdate { false };
  
+diff --git a/Source/WebCore/page/EventSource.cpp b/Source/WebCore/page/EventSource.cpp
+index 757765c3b4872d5a6f92b34e3f2ac67eaaf2dd82..89c1878afea0b81fcfd6fd1b44e307256cef7fbe 100644
+--- a/Source/WebCore/page/EventSource.cpp
++++ b/Source/WebCore/page/EventSource.cpp
+@@ -36,6 +36,7 @@
+ #include "CachedResourceRequestInitiators.h"
+ #include "ContentSecurityPolicy.h"
+ #include "EventNames.h"
++#include "InspectorInstrumentation.h"
+ #include "MessageEvent.h"
+ #include "ResourceError.h"
+ #include "ResourceRequest.h"
+@@ -93,6 +94,7 @@ EventSource::~EventSource()
+ 
+ void EventSource::connect()
+ {
++    InspectorInstrumentation::willSendEventSourceRequest(scriptExecutionContext());
```

Comment:
Can we instead set `inspectorInitiatorNodeIdentifier` or `inspectorInitiatorIdentifier` on the request created a few lines below and then check it inside the inspector?